tools/cli/flags.go

```diff
@@ -447,6 +447,48 @@ func getFlagsForScan() []cli.Flag {
 	}
 }
 
+func getFlagsForListArchived() []cli.Flag {
```

Comment:
Let extract some common flags from list commands and reuse. This will help people extend common feature.

Reply:
Done.